src/_assets/scss/style.scss

```diff
@@ -0,0 +1,162 @@
+$color-link: #5398e8;
+$color-text-base: #333;
+$l-header-width: 32vw;
+
+
+html {
+    box-sizing: border-box;
+    -ms-text-size-adjust: 100%;
+    -webkit-text-size-adjust: 100%;
+}
+
+*,
+*::before,
+*::after {
+    box-sizing: inherit;
+}
+
+html,
+body {
+    border: 0;
+    margin: 0;
+    padding: 0;
+    min-height: 100vh;
+}
+
+body {
+    color: $color-text-base;
+    display: flex;
+    font-family: Georgia, Times, serif;
+    font-size: 1.05em;
+    line-height: 1.5;
+}
+
+a {
+    color: $color-link;
+
+    &:hover,
+    &:focus {
+        text-decoration: none;
+    }
+}
+
+h1,
+h2,
+p,
+ol,
+ul {
+    margin: 0;
+}
+
+ul,
+ol {
+    list-style-position: outside;
+    margin-left: 0;
+    padding-left: 0;
+}
+
+h1 {
+    font-size: 2.1em;
+    margin-bottom: 1em;
+}
+
+
+/**
+ * Header.
+ */
+
+.l-header {
+    background-color: $color-text-base;
+    color: #fff;
+    display: flex;
+    flex-direction: column;
+    height: 100vh;
+    justify-content: center;
+    padding: 2em calc(2em + 71px) 2em 2.5em;
```

Comment:
I wanted to but then it won't work with `calc` due to the setting be a **pre-processor** thing.